quarkus/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.javaoperatorsdk</groupId>
+        <artifactId>jenvtest-project</artifactId>
+        <version>0.9.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jenvtest-quarkus-support</artifactId>
+    <packaging>jar</packaging>
+    <name>JEnvTest Quarkus Support</name>
+
+    <properties>
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+        <quarkus.platform.version>2.16.6.Final</quarkus.platform.version>
```

Comment:
We should probably target Quarkus 3 (which is already tagged).

Reply:
AFAIK there is no officeial release yet: https://github.com/quarkusio/quarkus/releases
neother on the website, also the automatic update script updates it into CR2